packs/elastic/pack.yaml

```diff
@@ -0,0 +1,10 @@
+---
+name : elastic
```

Comment:
We've discussed kwargs with manas, if you mean them. He thinks going with kwargs is just fine. Having as many classes as required in such a case will just an overkill.
Btw `get_client` has been fixed to use explicit parameters.